Tests/MapboxMapsTests/Ornaments/InfoButton/MapboxInfoButtonOrnamentViewTests.swift

```diff
@@ -0,0 +1,115 @@
+import XCTest
+@testable import MapboxMaps
+
+class MapboxInfoButtonOrnamentTests: MapViewIntegrationTestCase {
+    func testInfoButtonTapped() throws {
+        let infoButton = MapboxInfoButtonOrnament()
+        let parentViewController = MockParentViewController()
+        parentViewController.view.addSubview(infoButton)
```

Comment:
Looks like some of this setup is repeated in each test. Could you extract it into a setup method?